drivers/wifi/nrf700x/Kconfig

```diff
@@ -427,6 +427,15 @@ config NRF_WIFI_QOS_NULL_BASED_RETRIEVAL
 
 endchoice
 
+if NRF_WIFI_QOS_NULL_BASED_RETRIEVAL
+config NRF_WIFI_QOS_NULL_FRAME_INTERVAL_MS
```

Comment:
Shouldn't there be a `PS` in the name here somewhere?

Reply:
The naming is followed as per latest main. It can be changed later if required.